### DISCUSSION TITLE
Chit-Chat

### DISCUSSION THREAD

**Discussion by greggirwin** [red/red#4749] [General]
More social, less technical.

**Comment by hiiamboris** [comment]
spamspamspam ;)

**Reply by greggirwin** [reply]
I don't know if there's a spam bot. Manual intervention for now.

**Comment by koba-yu** [comment]
Seems more searchable than gitter 😊

**Reply by greggirwin** [reply]
Almost anything is. ;^)

**Comment by ghost** [comment]
Hi! :octocat:
